report/assets/styles.css

```diff
@@ -873,7 +873,6 @@
 
 .lh-audit--load-opportunity .lh-audit__display-text {
   text-align: right;
-  flex: 0 0 calc(3 * var(--report-font-size));
```

Comment:
This one potentially has the side effect of the right end of spark lines being misaligned between opportunities (without a fixed and reserved space for each)?

Reply:
![image](https://user-images.githubusercontent.com/683500/187307029-df084aad-c0a2-4af2-90d5-2469ced1b87e.png)

Reply:
Yeah good point, that's probably why this line was here in the first place. Bumping to `4 *` is probably best and then paul's suggestion just so a potential break doesn't happen in the middle of the word.